helm/hpcc/templates/thor.yaml

```diff
@@ -83,7 +83,7 @@ data:
             accessDali: "yes"
             accessEsp: "yes"
             helmVersion: 9.2.107-closedown0
-            {{- include "hpcc.addStandardLabels" (dict "root" $ "component" "eclagent" "name" "thor" "instance" $eclAgentJobName "instanceOf" (printf "%s-job" .eclAgentName)) | indent 8 }}
+            {{- include "hpcc.addStandardLabels" (dict "root" $ "component" "eclagent" "name" "thor" "instance" $eclAgentJobName "instanceOf" (printf "%s-job" .eclAgentName)) | indent 12 }}
```

Comment:
good catch! Meant template generation would have failed if eclAgentUseChildProcesses: false in Thor